libopenage/util/compiler.cpp

```diff
@@ -14,6 +16,10 @@ namespace util {
 
 
 std::string demangle(const char *symbol) {
+#ifdef _MSC_VER
+	// TODO: demangle names for MSVC;
+	return symbol;
```

Comment:
There is: [`UnDecorateSymbolName`](https://msdn.microsoft.com/en-us/library/windows/desktop/ms681400(v=vs.85).aspx). However, it has a problem mentioned in the Remarks section: it is single threaded. I can make this work with some synchronization primitives, but I'd prefer that to be in a subsequent PR, so the focus here remains on the build and we can get more opinions before going that way.
Suggestions?